### ISSUE TITLE
AAD not working when deployed to azurewebsites.

### ISSUE THREAD

**Issue by KNovotny** [projectkudu/slingshot#66]
Followed instructions, app works perfectly locally.  When deployed to azurewebsites, the app does not attempt to authenticate.  Everything looks to be set up properly.

**Comment by shrimpy** [comment]
I dont think you can deploy your own copy of slingshot, callback url (your site )  will not valid for authentication.

Thanks,
Xiaomin Wu

Sent from Outlook<http://taps.io/outlookmobile> Mobile

From: KNovotny
Sent: Wednesday, March 2, 7:36 AM
Subject: [slingshot] AAD not working when deployed to azurewebsites. (#66)
To: projectkudu/slingshot

Followed instructions, app works perfectly locally. When deployed to azurewebsites, the app does not attempt to authenticate. Everything looks to be set up properly.

-
